normandy/recipes/api/filters.py

```diff
@@ -14,6 +16,33 @@ def filter(self, qs, value):
         return qs
 
 
+class BaselineCapabilitiesFilter(django_filters.Filter):
+    """Filters recipe by whether they use only baseline capabilities, defaulting to only baseline."""
+
+    def __init__(self, *args, default_only_baseline=False, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.default_only_baseline = default_only_baseline
+
+    def filter(self, qs, value):
+        baseline_only = self.default_only_baseline
+
+        if value is not None:
+            lc_value = value.lower()
+            if lc_value in ["false", "0"]:
+                baseline_only = False
+
+        if baseline_only:
+            recipes = list(qs)
+            assert all(isinstance(recipe, Recipe) for recipe in recipes)
```

Comment:
note: assertions errors are not raised if the server runs with `python -O` (optimized)

Reply:
Good point. I'll change this to an exception.